utils/portspec/helpers.go

```diff
@@ -0,0 +1,52 @@
+package portspec
+
+import (
+	"sort"
+)
+
+const lastValidPort = 65535
+
+// GetUncoveredPortRanges returns non overlapping port ranges
+func GetUncoveredPortRanges(portSpecs ...*PortSpec) ([]*PortSpec, error) {
+
```

Comment:
Where is this used and why is it needed? I see it in the tests, but I am not sure where else is needed. Especially since there are different chains?

Reply:
I don't  that it is needed. Host mode ACLs have lower priority and they don't need to be split. They will only be hit if the Trireme ACLs are not hit. There is no need to go through this process of splitting ACLs when it can be done perfectly fine with priorities.